R/gh_fill.R

```diff
@@ -0,0 +1,28 @@
+#' Fill geohash prefix with members
```

Comment:
The package isn't currently using roxygen2, so you'll need to either write the .Rd directly, or convert the rest of the package to use roxygen2.